src/main/java/com/google/sps/servlets/Statistics.java

```diff
@@ -12,18 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.sps.servlets.utils;
+package com.google.sps.servlets;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class Statistics {
   // Aggregated score for a list of comments  
-  // Contains sentiment scores in the range [-1, 1] with 0.2 intervals.
-  private List<Double> aggregateValues;
+  // Contains number of sentiment scores in the range [-1, 1] with 0.2 intervals.
+  private List<Integer> aggregateValues;
```

Comment:
Should we also add getters for the private member variables included w/in this class in this PR?

Reply:
We discussed this as a pod, but the class is only used to encapsulate the data so it can be passed to js thru JSON